PhysicsTools/NanoAOD/python/custom_jme_cff.py

```diff
@@ -18,7 +18,7 @@
 from PhysicsTools.PatAlgos.tools.jetTools import supportedJetAlgos
 from PhysicsTools.PatAlgos.tools.jetTools import updateJetCollection
 
-import copy
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
```

Comment:
This one is not needed anymore, is it?

Reply:
I could not find any usage for it in the file (it's also removed in Marino's PR).

Reply:
I meant, `run3_common` is not used ;)

Reply:
oh yes sorry! good point I'll remove it!